### ISSUE TITLE
Make other properties modifiable using modifyEntry

### ISSUE THREAD

**Issue by travs** [melonproject/satellite#11]
Code [here](https://github.com/melonproject/satellite/blob/master/truffle/contracts/Satellite.sol#L74-L78) can only change an entry's URL property.
Ideally, there will be more properties that can be stored/changed, but this is dependent on how we end up implementing entry storage in the future as well (#10).

**Comment by retotrinkler** [comment]
Ok so just brainstorming
could be link to 
- github repository eg something like https://github.com/paritytech/contracts/blob/master/GithubHint.sol#L8-L12
- gist repo eg https://gist.github.com/anonymous/be21f66e6cf208ce78cbd2dece7e1db9 useful if u use browser solidity to dev contracts
- epm https://www.ethpm.com/registry - not quite sure yet how to best link epm

**Comment by travs** [comment]
@retotrinkler 
Yes, I suppose repositories could be hosted on many services, so maybe we could adopt a convention [similar to NPM's](https://docs.npmjs.com/files/package.json#repository) where they do `github:myuser/myrepo` or `bitbucket:example/repo`.
